src/main/resources/public/raml/resourceTypes/aggregated_object.raml

```diff
@@ -0,0 +1,28 @@
+description: This class returns aggregated data given a specific ID of an aggregated object.
```

Comment:
The class description needs to be updated now that it also contains the /query endpoint.